### ISSUE TITLE
identical conditions, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1363]
```abap
IF <ls_e>-line CP '#M#o#d#u#l#e*' OR <ls_e>-line CP '#m#o#d#u#l#e*'.
ENDIF.
```
